playbooks/wazo-acceptance/pre.yaml

```diff
@@ -23,15 +22,11 @@
     - name: Validate root key
       shell: "ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -o PreferredAuthentications=publickey root@localhost -n \"/bin/ls\""
 
-    - name: Install list of packages
+    - name: Install ansible
       become: yes
-      apt: name={{item}} state=installed
-      with_items:
-        - ansible
-        - libsasl2-dev
-        - linphone-nogtk
-        - python3-dev
-        - lsof
+      apt:
+        name: ansible
+        state: present
```

Comment:
isn't that supposed to be `state: installed`?

Reply:
@fredericlepied what is the inline syntax for a module? where is that documentation?
Also, the module apt do not have the state installed. Why it worked before? and old version of the module?

Reply:
No idea why it worked. The doc for modules is the regular Ansible documentation.

Reply:
Ah ok the *inline* syntax is just the regular yaml syntax :sweat_smile: